frontend/packages/container-security/src/components/image-vulnerability-utils.ts

```diff
@@ -0,0 +1,30 @@
+import { Vulnerability } from '../types';
+
+export enum VulnerabilitiesType {
+  allVulnerabilities = 'All vulnerabilities',
+  appDependency = 'App dependency',
+  baseImage = 'Base image',
+}
+
+const vulnerabilityMetadata = (vulnerability: Vulnerability) => {
+  return vulnerability.metadata ? JSON.parse(vulnerability.metadata) : vulnerability.metadata;
+};
+
+export const getVulnerabilityType = (vulnerability: Vulnerability) => {
+  const metadata = vulnerabilityMetadata(vulnerability);
+  return ['pypi', 'maven', 'npm'].includes(metadata?.RepoName) ||
+    metadata?.UpdatedBy === 'CodeReadyAnalytics'
+    ? VulnerabilitiesType.appDependency
+    : VulnerabilitiesType.baseImage;
```

Comment:
@invincibleJai , Actually `RepoName` would be right field.  `UpdatedBy == 'CodeReadyAnalytics'` shall be used to show attribution to Snyk, because all vulnerability from 'CodeReadyAnalytics' would be sourced from Snyk Intel.